apps/datahub/src/app/app.module.ts

```diff
@@ -181,6 +180,7 @@ export const metaReducers: MetaReducer[] = !environment.production ? [] : []
     }),
     OrganisationsComponent,
     LanguageSwitcherComponent,
+    NavigationBarComponent,
```

Comment:
Is that really needed? this component is part of the same app

Reply:
Ah right, it's not needed, this was a "make it standalone reflex".